org.eclipse.m2e.maven.runtime/pom.xml

```
@@ -20,7 +20,7 @@
 	</parent>
 
 	<artifactId>org.eclipse.m2e.maven.runtime</artifactId>
-	<version>3.9.101-SNAPSHOT</version>
+	<version>3.9.200-SNAPSHOT</version>
```

```suggestion
	<version>3.9.102-SNAPSHOT</version>
```
The hundreds of the micro version should correspond to the Maven release embedded into the m2e.maven.runtime. In cases like these, please just bump the last digit.

---

I thought 200 would indicate a significant change in the maven runtime, but whatever

---

Yes there is no optimal solution here, but if we hopefully manage to update to Maven 3.9.4 (https://github.com/eclipse-m2e/m2e-core/pull/1370) soon this question is obsolete anyway. :)